app/views/messages/index.html.erb

```diff
@@ -0,0 +1,76 @@
+<div class="container">
+  <%= link_to 'Home', root_path %>
+</div>
+
+<div class="container">
+<div class="row">
+<div class="col-8">
+  <h3>Messages</h3>
+  <div class="over">
+    <% @messages.each do |message| %>
+    <% if message.sender_id == current_user.id || message.recipient_id == current_user.id %>
+    <% if message.sender_id == current_user.id %>
+      <% recipient = message.recipient %>
+    <% else %>
+      <% recipient = message.sender %>
+    <% end %>
+    <table class="table tsize">
+        <col width="25%">
+        <col width="65%">
+        <col width="5%">
+        <col width="5%">
+        <tr>
+          <td><%= recipient.first_name %> <%= recipient.last_name %></td>
+          <td>
+            <% message.contents.each do |content| %>
+                <% if content.equal?(message.contents.last)  %>
+                  <%= content.body %>
+                <% end %>
+            <% end %>
+          </td>
+          <td>
+            <% if !message.unread_content_count(current_user).zero? %>
+              <%= message.unread_content_count(current_user) %>
+            <% end %>
+          <td>
+          <td><%= link_to '', message_contents_path(message), :class => "fas fa-pen-nib fa-xm" %></td>
+          <td><%= link_to '', message_path(message), method: :destroy, :class => "fas fa-trash-alt fa-xm" %></td>
+        </tr>
+    </table>
+   <% end %>
+  <% end %>
+  </div>
+</div>
+<div class="col-4">
+  <h3 class="people">Matched People</h3>
+  <div class="over">
+      <% @users.each do |user| %>
+      <% if user.id != current_user.id %>
+        <div class="container">
+          <div class="media border p-3">
+            <% if user.picture == true %>
+              <div>
+                <%= image_tag user.picture %>
+              </div>
+            <% else %>
+              <div>
+                <img class="face mr-3 mt-3 rounded-circle" src="https://images.unsplash.com/photo-1508109742312-c7d531211d11?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1050&q=80" alt="picture"> 
+              </div>
+            <% end %>
+            <div  class="media-body d-flex flex-column">
+              <div class= "p-2">
+                <div class="distance text-danger"><strong><%= user.distance.round(1) %> mi</strong></div>
+                <strong><%= user.first_name %> <%= user.last_name %></strong><br>
+                <%= user.email %><br>
+                <%= link_to "", messages_path(sender_id: current_user.id, recipient_id: user.id), method:'post', :class => "fas fa-paper-plane fa-xm" %>
+                <%= link_to "", messages_path(sender_id: current_user.id, recipient_id: user.id), method:'post', :class => "fas fa-search fa-xm" %>
```

Comment:
The text for both of these links are blank. Is that intentional?

Reply:
This empty strings are for font.